### ISSUE TITLE
Add example to package docs on pub.dev page.

### ISSUE THREAD

**Issue by UlyssesAlves** [vanlooverenkoen/moor_db_viewer#51]
https://pub.dev/packages/db_viewer

**Comment by vanlooverenkoen** [comment]
Because the db_viewer is the shared code between all db_viewer packages (moor_db_viewer, drift_db_viewer,...) this has low priority for now. Feel free to create a pull request.

**Comment by UlyssesAlves** [comment]
I couldn't grasp how it work actually, just by looking at the code. Maybe I haven't given enough time for it, but since [there's no SQLite support for it yet](https://github.com/vanlooverenkoen/db_viewer/issues/50) I'll have to wait for it to be available so that I can try it in my project, which uses SQLite. Only then I think I would be able to collaborate on the documentation for this project.

**Comment by vanlooverenkoen** [comment]
Yes indeed, for now you could use the moor & drift implementation as an example
